cita-chain/core/src/libchain/chain.rs

```diff
@@ -306,14 +307,19 @@ pub fn get_chain(db: &RocksDB) -> Option<Header> {
             &CurrentHash.get_index().to_vec(),
         )
         .unwrap_or(None)
-        .map(|h| H256::from_slice(&h));
+        .map(|h| decode::<H256>(&h));
 
     if let Some(hash) = res {
+        trace!("Get block height from hash : {:?}", hash);
+        let hash_key = (&hash as &DBIndex<BlockNumber, Item = H256>).get_index();
```

Comment:
maybe we should define a unified key-transfer level to use the same rule of key in future.

Reply:
okay